src/libra_py/dynamics/qtag/QTAG_config.py

```diff
@@ -0,0 +1,65 @@
+"""
```

Comment:
this should not be included in Libra itself, since this is a system-specific input - instead, please create a separate document (e.g. a Jupyter notebook to include as an example). Such a file would take care of all the system-specific input and would call the general functions we include in Libra